GameModes/StoryGameMode.cs

```diff
@@ -97,6 +98,23 @@ internal override void ResourceAvailable(OnlineResource onlineResource)
             }
         }
 
+        internal override void ResourceActive(OnlineResource onlineResource)
+        {
+            base.ResourceActive(onlineResource);
+            if (onlineResource is WorldSession ws)
+            {
+                var regionState = ws.world.regionState;
+                if (this.lobby.isOwner)
+                {
+                    ghostsTalkedTo = regionState.saveState.deathPersistentSaveData.ghostsTalkedTo.ToDictionary(kvp => kvp.Key.value, kvp => kvp.Value);
```

Comment:
Still want the debug here or was this just for testing?

Reply:
oops, forgot to remove